prober/prober.sh

```diff
@@ -0,0 +1,11 @@
+set -eEuo pipefail
+
+ID_TOKEN=$(curl "http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/default/identity?audience=$AUDIENCE" -H "Metadata-Flavor: Google")
+
+./jvsctl token --auth-token $ID_TOKEN -e "prober" > jvs_token
```

Comment:
Can we add some error logs?

Reply:
During our previous discussion with Chen, our idea is if the script failed, there will be a corresponding error log automatically written by cloud run job which will contain the specific error message. Therefore, there is no need to right error logs here.

Reply:
https://github.com/abcxyz/pmap/tree/main/cmd/policy

Reply:
The CLI's output is the error log

Reply:
I thought it still makes sense to throw error logs here so that when debug, at least we have some specific error messages we know to search.

Reply:
Is the error message here sufficient? https://github.com/abcxyz/jvs/blob/main/pkg/cli/validate.go#L166-L169

Reply:
I thought we will test both breakglass and nonbreakglass cases. Therefore, just the error messages from the CLI still looks confusing because I need to figure out the error message is from which CUJ. Since we won't test breakglass cases here, I don't think we need to add other error messages. If the job fail, I know that's for nonbreakglass case.

Reply:
Right, breakglass case should already be covered in unit test because it's self-contained.